frontend_tests/node_tests/narrow_activate.js

```diff
@@ -9,6 +9,9 @@ zrequire('narrow');
 zrequire('search_pill');
 zrequire('util');
 
+set_global('activity', {
+    is_initialized: function () { return false; },
+});
```

Comment:
It might be cleaner to just `zrequire('activity')` for this test.

Reply:
oh, okay, I'll change that then.

Reply:
We add `is_initialized` in this PR, so I'd have to stub it anyway...
Should I do that in `activity` and then zrequire?